tools/list_stl_dependencies.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+# Copyright (c) 2021 by Apex.AI Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+
+#=====================================================
+# usage:
+#   cd ICEORYX_ROOT_DIR
+#   ./tools/list_stl_dependencies.sh
+#=====================================================
+
```

Comment:
You could add something like `WORKSPACE=$(git rev-parse --show-toplevel)` to step automatically into the iceoryx base folder.

Reply:
Nice idea ... it is done!